packages/web/src/components/settings-sheet.tsx

```diff
@@ -240,7 +238,7 @@ function Error(props: { error: string | null }) {
     <div className="px-1.5 pb-1.5">
       <div className="bg-error text-error-foreground text-sm font-medium py-2 pl-[10px] pr-3 flex items-start gap-1.5 rounded-sm">
         <div className="shrink-0 mt-0.5">
-          <Info size={16} />
+          <Info size={16} className="shrink-0 mt-0.5" />
```

Comment:
Did we improve anything here? We should either remove the div around it or not do this? 

Make sure to test this one, I'm not sure it's equivalent behavior (all for it if it is)